tt_eager/tt_lib/csrc/tt_lib_bindings_tensor.cpp

```diff
@@ -674,14 +674,15 @@ void TensorModule(py::module& m_tensor) {
     // input embeddings
     m_tensor.def(
         "embeddings",
-        &embeddings,
+        py::overload_cast<const Tensor&, const Tensor&, bool, EmbeddingsType, std::optional<uint32_t>, const MemoryConfig&, std::optional<const DataType>, std::optional<Tensor> >(&embeddings),
```

Comment:
We don't need 2 bindings since queue id is an optional arg

Reply:
I have removed one bindings, Kindly review it once